src/app/tests/suites/certification/Test_TC_SMCO_2_2.yaml

```diff
@@ -0,0 +1,286 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name:
+    4.2.2. [TC-SMCO-2.2] Primary Functionality - Smoke Alarm with DUT as Server
+
+PICS:
+    - SMOKECO.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Smoke CO Alarm"
+    timeout: 990
+    TEST_EVENT_TRIGGER_KEY:
+        type: octet_string
+        defaultValue: "hex:00112233445566778899aabbccddeeff"
+    TEST_EVENT_TRIGGER_WARNING_SMOKE_ALARM:
+        type: int64u
+        defaultValue: "0xffffffff00000090"
+    TEST_EVENT_TRIGGER_CRITICAL_SMOKE_ALARM:
+        type: int64u
+        defaultValue: "0xffffffff0000009c"
+    TEST_EVENT_TRIGGER_SMOKE_ALARM_CLEAR:
+        type: int64u
+        defaultValue: "0xffffffff000000a0"
+
+tests:
+    - label: "Commission DUT to TH"
+      cluster: "DelayCommands"
+      endpoint: 1
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    # Get the latest eventNumber
+    - label: "Generate an event on the accessory"
+      cluster: "Unit Testing"
```

Comment:
This can't land in this form.

That said, "get the last event" is not very useful, since events can be getting generated continuously.  What do you need to know the last event number for?

Reply:
Without eventNumber, it's impossible to list all existing events in the readEvent's response. I've changed to setting eventNumber manually in config, is this ok?